planck-c/legal.c

```diff
@@ -27,7 +27,34 @@ void legal() {
     "not remove this notice, or any other, from this software.\n"
     "\n"
     "\n"
-  
+
+    "CLJS Bean\n"
+    "---------\n"
+    "\n"
+    "MIT License\n"
+    "\n"
```

Comment:
I suppose all of the other MIT licenses in this file don't actually include the full copy of the license.